blogposts/2024-07-18-running-rnw-from-a-mac.md

```diff
@@ -0,0 +1,88 @@
+
+Unlike Parallels, dev box runs in a cloud and does not have access to your local files, so you will need to redownload any source code that you need to get started developing. When you are finished with your development session and have shut down the virtual machine in the Dev Box portal, your machine's state will be saved for next time.
+
+![](assets/2024-07-18-running-rnw-from-a-mac/Screenshot-MicrosoftDevBox.jpg)
+
+## Takeaways about Microsoft Dev Box
+
+Microsoft Dev Box is a great tool when your team is ready for a larger level of investment in the Windows platform. Its feature offerings of cloud-based infrastructure, pre-configured dev boxes, and Intune secured workstations are great for enterprise-scale development.
+
+Microsoft Dev Box requires several [admin and configuration steps](https://learn.microsoft.com/azure/dev-box/quickstart-configure-dev-box-service) to get started, so if you're looking to quickly experiment with RNW within a VM, Parallels or another virtual machine offering may be a good option to get started with. Then once you're ready to scale your Windows VM usage across many developers to build large production experiences, Microsoft Dev Box could become a better option.
+
+## Overall Takeaways
+
+In this post, I looked at two options for building React Native for Windows apps from Mac devices using Windows VMs. Below is a table which captures the scenario comparisons between Microsoft Dev Box and Parallels.
+
+| Feature                                          | Microsoft Dev Box | Parallels |
+| ------------------------------------------------ | ----------------- | --------- |
+| Locally installed virtual machine software       |                   | ✅        |
+| Cloud-based virtual machine                      | ✅                |           |
+| Easy to follow documentation                     | ✅                | ✅        |
+| Great for quick experimentation with Windows     |                   | ✅        |
+| Great for enterprise development                 | ✅                |           |
+| Scale Windows VM usage across several developers | ✅                |           |
+| Pre-configured dev boxes                         | ✅                |           |
+
+I want to callout that, when your company is ready to invest in supporting Windows, having dedicated Windows hardware will be best developer experience.
+
+As you reach the end of reading, I encourage you to give React Native for Windows and Windows VMs a try! To get started with React Native for Windows visit [aka.ms/reactnative](https://microsoft.github.io/react-native-windows/). If you have any feedback or questions for the React Native for Windows team, you can reach out to us on GitHub at [microsoft/react-native-windows](https://github.com/microsoft/react-native-windows). We want to hear about your developer experience!
```

Comment:
```suggestion
I encourage you to give React Native for Windows and Windows VMs a try! To get started with React Native for Windows visit [aka.ms/reactnative](https://microsoft.github.io/react-native-windows/). If you have any feedback or questions for the React Native for Windows team, you can reach out to us on GitHub at [microsoft/react-native-windows](https://github.com/microsoft/react-native-windows). We want to hear about your developer experience!
```
Reading "As you reach the end of reading" definitely triggers my "too verbose-o-meter".